### PR TITLE
DOCSP-45195: Project fields

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-45195

### Staging Links

<li><a href=https://deploy-preview-105--docs-ruby.netlify.app/read/project>read/project</a></li><li><a href=https://deploy-preview-105--docs-ruby.netlify.app/read>read</a></li>

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/read/project.txt [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+option in a parameter to the ``find`` method. To set this option, use the
+following syntax:
+
+.. code-block:: ruby
+
+   { projection: { <field_name>: 0 } }
+
+The following example uses the ``find`` method to find all restaurants in which the ``name``
+field value is ``'Emerald Pub'``. Then, the code uses the ``projection`` option
+to instruct the find operation to omit the ``name`` and ``address`` fields
```
Comment (stephmarie17) [inline source/read/project.txt:127]: [i] The code example uses `grades` here, so can update description or code example to match.

File: source/read/project.txt [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Projection Types
+----------------
+
+You can use a projection to specify which fields to include in a return
+document, or to specify which fields to exclude. You cannot combine inclusion and
+exclusion statements in a single projection, unless you are excluding the
```
Comment (stephmarie17) [inline source/read/project.txt:50]: [s] Suggestion to reduce repetition:
```suggestion
You can use a projection to specify which fields to include or exclude in a return
document. You cannot combine inclusion and
exclusion statements in a single projection, unless you are excluding the
```

File: source/read/project.txt [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+``_id`` field.
+
+Specify Fields to Include
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To specify the fields to include in the result, specify the ``projection``
+option in a parameter to the ``find`` method. To set this option, use the following syntax:
```
Comment (stephmarie17) [inline source/read/project.txt:57]: [s] Can re-word a bit to avoid repetition of "specify", it's hard to do in this kind of doc but re-reading can make it hard to parse. Maybe something like:
```suggestion
To exclude specific fields from the result, specify the ``projection``
option in a parameter to the ``find`` method. To set this option, use the following syntax:
```

File: source/read/project.txt [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+   .. output::
+      :visible: false
+
+      {"borough"=>"Manhattan", "cuisine"=>"American", "name"=>"Emerald Pub"}
+      {"borough"=>"Queens", "cuisine"=>"American", "name"=>"Emerald Pub"}
+
+Specify Fields to Exclude
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To specify the fields to exclude from the result, specify the ``projection``
```
Comment (stephmarie17) [inline source/read/project.txt:117]: [s] Similar to the above, but just a suggestion!

**Review by stephmarie17 — APPROVED** [review]
LGTM with a few comments

File: source/read/project.txt (hunk 1 above)
Comment (norareidy) [reply source/read/project.txt:127]: oops thanks for catching!

**Review by comandeo-mongo — APPROVED** [review]
